src/main/java/core/levels/LevelManager.java

```diff
@@ -28,6 +28,10 @@
     private final OrthogonalTiledMapRenderer mapRenderer;
     private final WorldEntityManager entityManager;
     private final SpriteBatch batch;
+    static final int SPAWN_BUFFER = 15;
+    static final int VELOCITY_ITERATIONS = 6;
+    static final int POSITION_ITERATIONS = 2;
+    static final float MAX_STEP = 0.5f;
```

Comment:
only used in `step` method, either have var there or reference what it does in the javadoc for that method (currently mentions the specific value)

also, a comment to explain what this does would help